### PR TITLE
Securing tiller via running it locally.

### LINKED ISSUE #4852: Securing Helm documentation: add tillerless plugin
Hello,

The team I'm on wanted to not have an unauthenticated gRPC endpoint (that has access to change things) in our K8s cluster. We looked into [enabling TLS](https://github.com/helm/helm/blob/master/docs/securing_installation.md#the-tiller-grpc-endpoint-and-tls), but instead ended up using the Tillerless Helm v2 plugin (https://github.com/rimusz/helm-tiller).

FWIW, our opinion was we found it more desirable / easier (at least for us) to use the plugin, vs creating `*.tls` certs and setting up a pipeline to manage them (to store them, and create new ones when current ones expire - and this would've been especially cumbersome if we setup a [separate TLS certificate](https://github.com/helm/helm/blob/master/docs/securing_installation.md#best-practices-for-securing-helm-and-tiller) for each endpoint).
Also, whenever helm v3 comes out, the [proposal](https://github.com/helm/community/blob/master/helm-v3/000-helm-v3.md) calls for going tiller-less.

So I'd like to ask if it would make sense to update [securing_installation.md](https://github.com/helm/helm/blob/master/docs/securing_installation.md) to at least mention the plugin as another (if not the recommended) option (besides enabling TLS) for not having a wide open gRPC endpoint? 
If the answer is yes, I'd be happy to create such a PR to modify `securing_installation.md` - I just wanted to ask first. 
(By the way, currently the only place where this repo mentions the "Tillerless Helm v2" plugin is [docs/related.md](https://github.com/helm/helm/blob/master/docs/related.md).)

Thanks!

### PR DESCRIPTION
**Pull request by jolson490**
Resolves #4852

### CODE REVIEW

**Review by bacongobbler — CHANGES_REQUESTED** [review]
looks great! Just one small change and it's good to go.

File: docs/securing_installation.md [hunk 1]
```diff
@@ -81,32 +105,9 @@ Many very useful tools use the gRPC interface directly, and having been built ag
 
 ## Best Practices for Securing Helm and Tiller
 
-The following guidelines reiterate the Best Practices for securing Helm and Tiller and using them correctly. 
+The following guidelines reiterate the Best Practices for securing Helm and Tiller and using them correctly.
 
 1. Create a cluster with RBAC enabled
-2. Configure each Tiller gRPC endpoint to use a separate TLS certificate 
-3. Release information should be a Kubernetes Secret 
+2. Configure Tiller to not have an unauthenticated endpoint
```
[the remark this answers is not in the document]
Comment (jolson490) [reply docs/securing_installation.md]: no probelm - done.

**Comment by mattfarina** [comment]
@jolson490 It appears you missed the signoff on one of the two commits. A quick fix set of commands can be found on the checks tab at https://github.com/helm/helm/pull/4869/checks.

**Review by bacongobbler — APPROVED** [review]

**Comment by bacongobbler** [comment]
thanks!

**Pull request merged**